l10n_ch_base_bank/tests/test_search_invoice.py

```diff
@@ -29,14 +29,15 @@ def setUp(self):
             'type': 'bank',
             'code': 'BNK42',
             'bank_id': bank.id,
-            'bank_acc_number': '01-1234-1',
+            'bank_acc_number': '10-8060-7',
```

Comment:
Why do this value change?

Reply:
bank_acc_number should be unique, when I'd relapsed on create it's start failing from which point I don't remeber

Reply:
ok, I restored the initials numbers (before my fix)